### DISCUSSION TITLE
"Prefer downloads without video" should probably apply to import too

### DISCUSSION THREAD

**Discussion by Sublimelime** [ppy/osu#17760] [Ideas] [answered]
The setting that makes beatmaps download without video should also make the client discard the video component of beatmaps imported as well. Currently if you import an osz file with a video it'll import the video as well, which consumes extra space (the very thing you're trying to avoid with the prefer-no-vid setting). Ideally when this setting is set it should just ignore any video files during import.

The main way I'm encountering this pain point is with beatmap packs (which come with videos) but I imagine this also affects people directly sharing osz files or importing from stable.

**Comment by peppy** [comment, marked as answer]
I think we'd sooner have a "Delete all videos" option than this. If you are importing a file, you expect it to import as is.
